packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -5,113 +5,167 @@ import {
+    `
   }
+  color: ${props => props.selected ? props.theme.global.colors['neutral-6'] : props.theme.global.colors[props.theme.dark ? 'neutral-6' : 'neutral-7']};
 
-  span {
-    text-align: ${props => props.columnsCount === 3 ? 'center' : 'left'};
+  &:focus {
+    border: 2px solid ${props => props.theme.global.colors['accent-1']};
+    outline: none;
+  }
+  
+  &:hover {
+    background: ${props => props.theme.global.colors['accent-1']};
+    border: 2px solid ${props => props.theme.global.colors['accent-1']};
+    box-shadow: 0 0 8px 2px ${props => props.theme.global.colors['accent-1']};
+    color: ${props => props.theme.global.colors['neutral-7']};
   }
+`
 
-  @media (768px < width < 1280px) {
-    flex-direction: ${props => props.columnsCount === 1 ? 'row' : 'column'};
+const StyledButton = styled(Button)`
+  flex-grow: 1;
+  height: 100%;
+`
 
-    img {
-      margin: ${props => props.columnsCount === 1 ? '0 1ch 0 0' : '0'};
-    }
+const StyledImage = styled(Image)`
+  border-radius: 4px;
+  object-fit: cover;
+`
 
-    span {
-      text-align: ${props => props.columnsCount === 1 ? 'left' : 'center'};
-    }
-  }
+const StyledLabel = styled(Text)`
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  overflow: hidden;
 `
 
+const DEFAULT_HANDLER = () => true
+
 function ChoiceButton({
-  ariaChecked = undefined,
   choiceId = '',
   choiceLabel = '',
-  columnsCount = 1,
   disabled = false,
   hasFocus = false,
-  onChoose = () => true,
-  onKeyDown = () => true,
-  role='button',
+  onChoose = DEFAULT_HANDLER,
+  onDelete = DEFAULT_HANDLER,
+  onKeyDown = DEFAULT_HANDLER,
   selected = false,
+  shadedBackground = false,
   src = '',
   tabIndex = -1,
   thumbnailSize = 'none'
 }) {
-  const choiceButton = useRef(null)
+  const choiceMenuItem = useRef(null)
   const handleClick = useCallback(() => {
     onChoose(choiceId)
   }, [choiceId, onChoose])
+  const handleDelete = useCallback(() => {
+    onDelete(choiceId)
+  }, [choiceId, onDelete])
   const handleKeyDown = useCallback((event) => {
     onKeyDown(choiceId, event)
   }, [choiceId, onKeyDown])
 
   useEffect(() => {
-    if (choiceButton && hasFocus) {
-      choiceButton.current.focus()
+    if (choiceMenuItem && hasFocus) {
+      choiceMenuItem.current.focus()
     }
   })
 
-  let thumbnailHeight = 0
-  if (thumbnailSize === 'small') {
-    thumbnailHeight = 21
-  }
-  if (thumbnailSize === 'medium') {
-    thumbnailHeight = 42
-  }
-  if (thumbnailSize === 'large') {
-    thumbnailHeight = 84
-  }
-
+  const thumbnailHeight = 150
   const thumbnailWidth = Math.round(thumbnailHeight * THUMBNAIL_ASPECT_RATIO)
   const thumbnailSrc = `https://thumbnails.zooniverse.org/${thumbnailWidth}x${thumbnailHeight}/${src.slice(8)}`
   
   return (
-    <Button
-      ref={choiceButton}
-      aria-checked={ariaChecked}
-      disabled={disabled}
+    <StyledBox
+      ref={choiceMenuItem}  
+      // TODO: add the following to translations
```

Comment:
Also want to keep these translations TODOs for now?

Reply:
Added translations